de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRImportBL.java

```diff
@@ -1248,6 +1232,26 @@ public void unlinkESRImportLinesFor(final I_C_BankStatementLine_Ref bankStatemen
 		}
 	}
 
+	public void runESRImportFor(final RunESRImportCriteria runESRImportCriteria)
```

Comment:
Instead of `runESRImportFor` woundering if it wouldn't be better to call it `scheduleESRImport`.
`schedule` because it's gonna happen async.